ecmascript/dep-graph/src/lib.rs

```diff
@@ -9,7 +9,7 @@ use swc_ecma_visit::{self, Node, Visit, VisitWith};
 
 pub fn analyze_dependencies(
     module: &ast::Module,
-    comments: &impl Comments,
+    comments: &dyn Comments,
```

Comment:
Are there performance implications for doing this? Maybe we should leave it as impl? I'm ok either way.